services/wfm/src/main/java/org/openkilda/wfm/topology/reroute/bolts/FlowThrottlingBolt.java

```diff
@@ -77,8 +70,9 @@ protected void doTick(Tuple tuple) {
 
     @Override
     protected void doWork(Tuple tuple) {
-        reroutesThrottling.putRequest(tuple.getStringByField(CacheBolt.FLOW_ID_FIELD),
-                tuple.getStringByField(CacheBolt.CORRELATION_ID_FIELD));
+        logger.info("Incoming reroute {}", tuple.getStringByField(RerouteBolt.FLOW_ID_FIELD));
```

Comment:
There is no need of such log message, because it writes down almost the same as reroutesThrottling.putRequest

Reply:
This is fixed.